WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeAccountChooserListAdapter.java

```diff
@@ -82,4 +82,14 @@ public ViewHolder(View view) {
     public interface OnPublicizeAccountChooserListener {
         void onAccountSelected(int selectedIndex);
     }
+
+    private String getName(PublicizeConnection connection) {
+        String name = connection.getExternalDisplayName();
+
+        if (name.isEmpty()) {
```

Comment:
Let's use `TextUtils.isEmpty(name)` here so we also have `null` as a possibility